optashift-employee-rostering-gwtui/pom.xml

```diff
@@ -102,6 +102,27 @@
           <moduleName>org.optaplanner.openshift.employeerostering.gwtui.WorkerRosteringWebapp</moduleName>
         </configuration>
       </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-resources-plugin</artifactId>
+        <version>2.3</version>
+      </plugin>
     </plugins>
+    <resources>
+      <resource>
+        <directory>src/main/java</directory>
+        <includes>
+          <include>**/*.html</include>
+          <include>**/*.css</include>
+        </includes>
+      </resource>
+      <resource>
+        <directory>src/main/resources</directory>
+        <includes>
+          <include>**/*.properties</include>
```

Comment:
this means that all non properties files aren't resources any more?
What if I have a png image resource?

Reply:
Good point, probably should change to **/* (I think maven wasn't copying resources after adding the src/main/java as a resource directory)

Reply:
Specifying no includes should do the trick too, and it's less code to maintain (less configuration by exception, so less error prone too)